### PR TITLE
docs: update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by acateland**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
No documentation about coding style to follow

**What is the new behavior?**
added a section on the contributing.MD with link to wiki page.

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

**Other information**:

### CODE REVIEW

**Comment by talend-glorieux** [comment]
Please fill the pull request template.

**Comment by acateland** [comment]
i din't yet because its not done and ready for review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -111,3 +112,7 @@ The footer should contain
 
 **Breaking Changes** should start with the word `BREAKING CHANGE:` with a space
 or two newlines. The rest of the commit message is then used for this.
+
+## <a name="codestyle"></a> Code style
```
Comment (talend-glorieux) [inline CONTRIBUTING.md]: No need for anchor.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -111,3 +112,7 @@ The footer should contain
...
+
+https://github.com/Talend/ui/wiki/Javascript-coding-style
```
Comment (talend-glorieux) [inline CONTRIBUTING.md]: Add all the other code style pages.

**Pull request merged**